Desktop/modules/dynamicmodule.cpp

```diff
@@ -470,12 +470,12 @@ void DynamicModule::unpackage()
 
 std::string DynamicModule::getLibPathsToUse()
 {
-	std::string libPathsToUse = "c('" + shortenWinPaths(moduleRLibrary()).toStdString()	+ "')";
+	std::string libPathsToUse = "c('" + AppDirs::rHome().toStdString() + "/library', '" + shortenWinPaths(moduleRLibrary()).toStdString()	+ "')";
 
-	std::vector<std::string> requiredLibPaths = fq(DynamicModules::dynMods()->requiredModulesLibPaths(tq(_name)));
+	//std::vector<std::string> requiredLibPaths = fq(DynamicModules::dynMods()->requiredModulesLibPaths(tq(_name)));
 
-	for(const std::string & path : requiredLibPaths)
-		libPathsToUse += ", '" + path + "'";
+	//for(const std::string & path : requiredLibPaths)
+	//	libPathsToUse += ", '" + path + "'";
```

Comment:
does this remove the dependency among modules? So the libPaths of e.g., jaspAnova become

Modules/jaspAnova
R/library

?

Reply:
No that is only solved in `development` (the old deps getting loaded and thus messing up new dynamic modules installed later)